extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -32,6 +32,7 @@ import {user} from '../../../../src/log';
 
 describes.realWin('amp-story-bookend', {amp: true}, env => {
   let win;
+  let document;
```

Comment:
`document` is a reserved word in JavaScript, to be honest I wouldn't know if you're actually re-assigning the `document` or if it's still keeping its original value.

Reply:
Nit: [it's not](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords). 😉

Reply:
Ah, next time I shouldn't look up JavaScript things on w3schools :eyes:

Reply:
Did `window` `document` etc stop being reserved when they introduced block scope variables?

Reply:
In root level blocks, yes. But they were never reserved (since ES5ish at least, when I started following the spec) with inside function blocks:

```js
function foo(document) {
  return document;
}

function bar() {
  var document = 1;
  return document;
}

foo(1) === bar();
```